### DISCUSSION TITLE
Code section highlighting with #%%

### DISCUSSION THREAD

**Discussion by joaofpugliese** [REditorSupport/languageserver#666] [General]
Hi all, I have been using R on vscode for a while. Sorry if this is the wrong place to post.

Until recently, I had been using code section delimiters with

```
#%%
print(test)
#%%
```

This created a highlighted background color with a "Run chunk" option.

Recently, however, something changed and now chunks defined simply with a single #%%, as in

```
#%%
print(test)
```

And the highlighted background color is gone. 

I can't find any discussion of this. Sorry if this is the wrong place to post.

**Comment by ThomasSoeiro** [comment]
This is defined in the vscode-R extension, not in the languageserver package.

https://github.com/REditorSupport/vscode-R/pull/662 and https://github.com/REditorSupport/vscode-R/pull/1455
